src/main/java/com/school/school/subjects/SubjectController.java

```diff
@@ -13,14 +25,35 @@ public class SubjectController {
     SubjectService subjectService;
 
     @GetMapping
-    public List<Subject> list()
-    {
+    public List<Subject> list() {
         return subjectService.list();
     }
 
+    @GetMapping("/{deleted}")
```

Comment:
@GetMapping() - returns all not deleted
@GetMapping("/deleted") - returns all deleted